system/assets/js/admin/presidecore/preside.uber.select.js

```diff
@@ -365,10 +365,21 @@
 				if ( this.local_options[i].empty ) { this.local_options.splice( i, 1 ); }
 			}
 
+			var prefetch_url = this.form_field.getAttribute( "data-prefetch-url" );
+			var remote_url   = this.form_field.getAttribute( "data-remote-url" );
```

Comment:
in our *js*, we'd like `var` definitions at the _top_ of functions and using the style:

```
var variable1 = true
  , variable2 = false
  , variable3, variable 4; // not yet set vars
```

These vars should be moved up aroud line 355 (and declared in the list style).